docs/source/gbd2017_models/risk_attributable_causes/chronic_kidney_disease/index.rst

```diff
@@ -148,11 +148,10 @@ Mapping CKD States to IKF Categories in Vivarium
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
-.. todo::
+* EMR = 0 for all states, except for Stage V.
+
+* Stage V is the only Stage which is fatal, where there will be YLLs and Deaths.
 
-  Describe the clinical and mathematical assumptions made for this cause model,
-  and the limitations these assumptions impose on the applicability of the
-  model.
 
```

Comment:
Presently, we are using prevalence for each stage of CKD to assign the each person in the population a CKD severity state.  We are assuming (for now) that there is no transition between states.  As a result, we should expect the prevalence for early stage CKD to swell as the simulation runs, since there is nowhere for these new incident cases to go.  If we had remission (which is actually the term used for "progression") rates, we could use these as inter-stage incidence rates.